src/core_modules/capture-core/components/CardList/CardListItem.component.js

```diff
@@ -1,117 +1,201 @@
-                    </div>
+                <div className={classes.itemValuesContainer}>
+                    <Grid container spacing={2}>
+                        {
+                            imageDataElement &&
+                            <Grid item>
+                                {renderImageDataElement(imageDataElement)}
+                            </Grid>
+                        }
+                        <Grid item xs={12} sm container>
+                            <Grid item xs container direction="column" spacing={2}>
+                                {
+                                    dataElements.map(element => (
+                                        <Grid item xs>
+                                            <div key={element.id} className={classes.value}>
+                                                <span className={classes.elementName}>
+                                                    {element.name}:&nbsp;
+                                                </span>
+                                                <span className={classes.elementValue}>
+                                                    {item.values[element.id]}
+                                                </span>
+                                            </div>
+                                        </Grid>
+                                    ))
+                                }
+                            </Grid>
+                            <Grid item>
+                                {
+                                    item.tei && item.tei.lastUpdated &&
+                                    <div className={classes.lastUpdated}>
+                                        { i18n.t('Last updated') } {item.tei && moment(item.tei.lastUpdated).fromNow()}
+                                    </div>
+                                }
+                                <div className={classes.enrolled}>
+                                    {
+                                        (enrollmentStatus === enrollmentStatuses.ACTIVE) &&
+                                        <Tag
+                                            dataTest="dhis2-uicore-tag"
+                                            positive
+                                            icon={
+                                                <DoneIcon
+                                                    style={{
+                                                        transformBox: 'view-box',
+                                                        fontSize: 14,
+                                                        position: 'relative',
+                                                        top: '-1px',
+                                                    }}
+                                                />
+                                            }
+                                        >
+                                            {i18n.t('Enrolled: Active')}
```

Comment:
Had a brief chat with @Bekkalizer: Badge text suggestions:

- enrollmentStatuses.ACTIVE: "Enrolled"
- enrollmentStatuses.CANCELLED: "Previously enrolled"
- enrollmentStatuses.COMPLETED: "Previously enrolled"

Reply:
Makes sense. Updating the text